File: 프로그래머스/unrated/181883. 수열과 구간 쿼리 1/수열과 구간 쿼리 1.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    for (int i = 0; i < queries.size(); i++)
    {
        int from = queries[i][0];
        int to = queries[i][1];
        for (int j = from; j <= to; j++)
            arr[j]++;
    }
    return arr;
}

File: 프로그래머스/0/181883. 수열과 구간 쿼리 1/수열과 구간 쿼리 1.java
import java.util.Arrays;
import java.util.stream.IntStream;

class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        Arrays.stream(queries)
                .forEach(query -> IntStream.rangeClosed(query[0], query[1])
                        .forEach(j -> arr[j]++)
                );
        return arr;
    }
}